dart/test/mocks/mock_environment_variables.dart

```diff
@@ -5,29 +5,29 @@ import 'no_such_method_provider.dart';
 class MockEnvironmentVariables extends EnvironmentVariables
     with NoSuchMethodProvider {
   MockEnvironmentVariables({
-    String? dist,
-    String? dsn,
-    String? environment,
-    String? release,
+    required String dist,
+    required String dsn,
+    required String environment,
+    required String release,
   })  : _dist = dist,
         _dsn = dsn,
         _environment = environment,
         _release = release;
 
-  final String? _dist;
-  final String? _dsn;
-  final String? _environment;
-  final String? _release;
+  final String _dist;
+  final String _dsn;
+  final String _environment;
+  final String _release;
 
   @override
-  String? get dist => _dist;
+  String get dist => _dist;
 
   @override
-  String? get dsn => _dsn;
+  String get dsn => _dsn;
 
   @override
-  String? get environment => _environment;
+  String get environment => _environment;
 
   @override
-  String? get release => _release;
+  String get release => _release;
```

Comment:
I'd not change this otherwise you break the contract of the interface EnvironmentVariables